kb_uploadmethods.spec

```diff
@@ -361,6 +361,35 @@ module kb_uploadmethods {
           returns (UploadMethodResult returnVal) authentication required;
 
   /*
+  required params:
+  model_file: subdirectory file path for model file
+  e.g.
+    for file: /data/bulk/user_name/file_name
+    staging_file_subdir_path is file_name
+    for file: /data/bulk/user_name/subdir_1/subdir_2/file_name
+    staging_file_subdir_path is subdir_1/subdir_2/file_name
+  compounds_file: same as above for compound (only used for tsv)
+  file_type: one of "tsv", "excel", "sbml"
+  genome: the associated species genome
+  biomasses: one or more biomass reactions in model
+  model_name: output FBAModel object name
+  workspace_name: workspace name/ID of the object
+  */
+
+  typedef structure {
+    string model_file;
+    string compounds_file;
+    string file_type;
+    string genome;
+    string biomass;
+    string model_name;
+    workspace_name workspace_name;
+  } FileToFBAModelParams;
+
+  funcdef import_file_as_fba_model_from_staging(FileToFBAModelParams params)
+        returns (UploadMethodResult returnVal) authentication required;
```

Comment:
So UploadMethodResult won't result error. But UploadMethodResult has obj_ref, but your func is returning ref.

Reply:
I think it make most sense to match the rest of the functions here and return the same type so I'll rename to ref and make sure I return a report